projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -370,6 +370,33 @@ def is_vectorizer_disabled_query(self) -> sql.Composed:
             self.vectorizer_table_ident,
         )
 
+    @cached_property
+    def requeue_or_remove_work_query(self) -> sql.Composed:
```

Comment:
I would love this function to be documented, mentioning we implement linear backoff, etc 🙏 WDYT?

Reply:
added, thanks!